src/test/java/de/numcodex/feasibility_gui_backend/model/StructuredQueryV2ParserTest.java

```diff
@@ -0,0 +1,152 @@
+          }, {
+            "termCode" : [ {
+              "code" : "F09",
+              "system" : "http://fhir.de/CodeSystem/dimdi/icd-10-gm",
+              "display" : "F09"
+            } ],
+            "timeRestriction" : {
+              "beforeDate" : "2021-10-09",
+              "afterDate" : "2021-09-09"
+            }
+          } ] ],
+          "exclusionCriteria" : [ [ {
+            "termCode" : [ {
+              "code" : "LL2191-6",
+              "system" : "http://loinc.org",
+              "display" : "Geschlecht"
+            } ],
+            "valueFilter" : {
+              "type" : "concept",
+              "selectedConcepts" : [ {
+                "code" : "male",
+                "system" : "",
+                "version" : "",
+                "display" : "male"
+              } ]
+            }
+          } ], [ {
+            "termCode" : [ {
+              "code" : "30525-0",
+              "system" : "http://loinc.org",
+              "display" : "Alter"
+            } ],
+            "valueFilter" : {
+              "type" : "quantity-comparator",
+              "comparator" : "gt",
+              "unit" : {
+                "code" : "year",
+                "display" : "Jahr"
+              },
+              "value" : 65.0
+            }
+          } ], [ {
+            "termCode" : [ {
+              "code" : "F00.9",
+              "system" : "http://fhir.de/CodeSystem/dimdi/icd-10-gm",
+              "display" : "F00.9"
+            } ]
+          }, {
+            "termCode" : [ {
+              "code" : "8310-5",
+              "system" : "http://loinc.org",
+              "display" : "Körpertemperatur"
+            } ],
+            "attributeFilters" : [ {
+              "type" : "concept",
+              "selectedConcepts" : [ {
+                "code" : "LA9370-3",
+                "system" : "http://loinc.org",
+                "display" : "Axillary"
+              } ],
+              "attributeCode" : {
+                "code" : "method",
+                "system" : "abide",
+                "display" : "method"
+              }
+            } ],
+            "valueFilter" : {
+              "type" : "quantity-range",
+              "unit" : {
+                "code" : "Cel",
+                "display" : "°C"
+              },
+              "minValue" : 35.0,
+              "maxValue" : 39.0
+            },
+            "timeRestriction" : {
+              "beforeDate" : "2021-10-09",
+              "afterDate" : "2021-09-09"
+            }
+          } ] ],
+          "display" : "Beispiel-Query"
+        }""";
+
+    var objectMapper = new ObjectMapper();
+    var test = objectMapper.readValue(testQuery, StructuredQuery.class);
+    var jsonString = objectMapper.writeValueAsString(test);
+    assert (jsonString.contains("attributeCode"));
+    assert (jsonString.contains("exclusionCriteria"));
+    assert (jsonString.contains("timeRestriction"));
+    assert (jsonString.contains("attributeCode"));
```

Comment:
```suggestion
    assert (jsonString.contains("attributeFilters"));
```

Could be a potential typo, otherwise this line would be a duplicate.